### PR TITLE
Doug update transcribe code example

### PR DESCRIPTION
**Pull request by Doug-AWS**
## Motivation and Context
Transcribe code example had hard-coded filename.
There was no readme.

## Description
Added -a AUDIO-FILE command-line option, along with -r REGION and -v (verbose).

## Testing
I ran clippy and tested it with and without the required -a option.

## Checklist
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated `aws/SDK_CHANGELOG.md` if applicable

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws/sdk/examples/transcribestreaming/src/main.rs [hunk 1]
```diff
@@ -4,27 +4,72 @@
...
+    verbose: bool,
+}
 
 const CHUNK_SIZE: usize = 8192;
 
+/// Transcribes an audio file to text.
+/// # Arguments
+///
+/// * `-a AUDIO_FILE` - The name of the audio file.
+///   It must be a WAV file.
```
Comment (jdisanti) [inline aws/sdk/examples/transcribestreaming/src/main.rs:37]: Probably good to call out that this is a limitation of the example and not of Transcribe.

File: aws/sdk/examples/transcribestreaming/src/main.rs [hunk 2]
```diff
@@ -60,14 +107,14 @@ async fn main() -> Result<(), Error> {
...
 
     Ok(())
 }
 
-fn pcm_data() -> Vec<u8> {
-    let audio = include_bytes!("../audio/hello-transcribe-8000.wav");
-    let reader = hound::WavReader::new(&audio[..]).unwrap();
+fn pcm_data(audio_file: &str) -> Vec<u8> {
+    //let audio = include_bytes!("../audio/hello-transcribe-8000.wav");
+    //let reader = hound::WavReader::new(&audio[..]).unwrap();
```
Comment (jdisanti) [inline aws/sdk/examples/transcribestreaming/src/main.rs:116]: Did you intend to remove these?
```suggestion
```

**Review by jdisanti — APPROVED** [review]
Looks good. Just a couple minor comments.

File: aws/sdk/examples/transcribestreaming/src/main.rs (hunk 1 above)
Comment (Doug-AWS) [reply aws/sdk/examples/transcribestreaming/src/main.rs:37]: Good point. I'll add a note that mentions the supported formats.

File: aws/sdk/examples/transcribestreaming/src/main.rs (hunk 2 above)
Comment (Doug-AWS) [reply aws/sdk/examples/transcribestreaming/src/main.rs:116]: Oops, yes.

**Review by jdisanti — COMMENTED** [review]
These changes look good. Could you update the SDK changelog?

**Comment by Doug-AWS** [comment]
> These changes look good. Could you update the SDK changelog?

Done

**Review by jdisanti — APPROVED** [review]

**Pull request merged**